__tests__/test-utils.js

```diff
@@ -47,6 +47,9 @@ class TestUtils {
     "Weg6WEncAEJs5qDbGUxcLTR3iycM3hrt4C",
     "WSVarF73e6UVccGwb44FvTtqFWsHQmjKCt",
     "Wc5YHn861241iLY42mFT8z1dT1UdsNWkfs",
+    "WU6KGNPuaRG4VCKC1fsKDh4fRZGbkqxG8Y",
```

Comment:
Why these new addresses?

Reply:
Since I had to use more of the wallet's addresses on the new tests, any query that involves the _gap limit_ also fetched more addresses that were not specified before on the mocked code.

For example, this `addresses` test:
https://github.com/HathorNetwork/hathor-wallet-headless/blob/ca9f1938522cfe8ee86e6f712e3d2224f53215a3/__tests__/addresses.test.js#L28

This is part of the refactoring I had to do on all the mocked tests fixtures on commit https://github.com/HathorNetwork/hathor-wallet-headless/pull/160/commits/e8f39488a88d573a63c581d6d9d8734f8fa1b08c